frontend/pages/policies/ManagePoliciesPage/components/PoliciesTable/PoliciesTable.tests.tsx

```diff
@@ -34,7 +34,41 @@ describe("Policies table", () => {
       />
     );
 
-    expect(screen.getByText("You don't have any policies")).toBeInTheDocument();
+    expect(
+      screen.getByText("You don't have any policies that apply to all teams")
+    ).toBeInTheDocument();
+    expect(screen.queryByText("Name")).toBeNull();
+  });
+
+  it("Renders the page-wide empty state when no policies are present (specific team)", async () => {
+    const render = createCustomRenderer({
+      context: {
+        app: {
+          isGlobalAdmin: true,
+          currentUser: createMockUser(),
+        },
+      },
+    });
+
+    render(
+      <PoliciesTable
+        policiesList={[]}
+        isLoading={false}
+        // eslint-disable-next-line @typescript-eslint/no-empty-function
+        onDeletePolicyClick={() => {}}
```

Comment:
I think cleaner to use `jest.fn()` here, which doesn't need the typescript override. Looks like some
guy named Jac Obshandling really blew it by starting this pattern above, but we can forgive him.

Reply:
or `noop`

Reply:
replaced all empty fns with `noop`, good call